system/ee/installer/updates/ud_3_01_00.php

```diff
@@ -379,6 +379,7 @@ private function synchronize_layouts()
 
         $layouts = ee('Model')->get('ChannelLayout')
             ->with('Channel')
+            ->fields('field_layout', 'Channel.field_group')
             ->all();
```

Comment:
Is this change unrelated to cloning?

Reply:
I am not 100% sure at this point. It is related in general to the line of changes we've agreed on where we only fetch the data that we need during the update process